Assets/Failsafe/UI/MenuSettings/GraphicsMenuSettings.cs

```diff
@@ -0,0 +1,354 @@
+
+    void Start()
+    {
+        postProcessVolume.profile.TryGetSettings(out motionBlur);
+        postProcessVolume.profile.TryGetSettings(out bloom);
+
+
+        
+        InitializeScreenModeDropdown();
+        InitializeVSyncToggle();
+        InitializeMotionBlurToggle();
+        InitializeBloomToggle();
+        InitializeMaxFPSDropdown();
+        InitializeShadowDropdown();
+        InitializeModelDropdown();
+        StartResolution();
+    }
+
+
+    void StartResolution()
+    {
+        resolution.options.Clear();
+        List<string> options = new List<string>();
+        resolutions = Screen.resolutions;
+        int currentResolutionIndex = 0;
+
+        for (int i = 0; i < resolutions.Length; i++)
+        {
+            string option = resolutions[i].width + "x" + resolutions[i].height + " " + resolutions[i].refreshRateRatio + "Hz";
+            options.Add(option);
+            if (resolutions[i].width == Screen.currentResolution.width && resolutions[i].height == Screen.currentResolution.height)
+                currentResolutionIndex = i;
+
+
+        }
+        
+        resolution.AddOptions(options);
+
+        LoadSettings(currentResolutionIndex);
+
+    }
+
+    private void InitializeVSyncToggle()
+    {
+        
+        vSyncToggle.isOn = QualitySettings.vSyncCount > 0;
+        vSyncToggle.onValueChanged.AddListener(SetVSync);
+    }
+
+    private void InitializeMotionBlurToggle()
+    {
+        
+        motionBlurToggle.isOn = motionBlur != null && motionBlur.active;
+        motionBlurToggle.onValueChanged.AddListener(SetMotionBlur);
+    }
+
+    private void InitializeBloomToggle()
+    {
+        
+        bloomToggle.isOn = bloom != null && bloom.active;
+        bloomToggle.onValueChanged.AddListener(SetBloom);
+    }
+
+    public void SetVSync(bool isVSync)
+    {
+        QualitySettings.vSyncCount = isVSync ? 1 : 0;
+        Debug.Log($"VSync is {(isVSync ? "ON" : "OFF")}");
+        
+    }
+
+    public void SetMotionBlur(bool isMotionBlur)
+    {
+        if (motionBlur != null)
+        {
+            motionBlur.active = isMotionBlur;
+            Debug.Log($"Motion Blur {(isMotionBlur ? "ON" : "OFF")}");
+        }
+
+
+    }
+
+    public void SetBloom(bool isBloom)
+    {
+        if (bloom != null)
+        {
+            bloom.active = isBloom;
+            Debug.Log($"Bloom {(isBloom ? "ON" : "OFF")}");
+        }
+
+
+    }
+
+
+    private void InitializeScreenModeDropdown()
+    {
+        screenModeDropdown.options.Clear();
+        List<string> options = new List<string> {
+            "Fullscreen",
+            "Borderless",
+            "Windowed"
+        };
+
+        screenModeDropdown.AddOptions(options);
+        SetCurrentScreenModeIndex(); 
+
+    }
+    
+    private void InitializeMaxFPSDropdown()
+    {
+        MaxFPS.options.Clear();
+        List<string> options = new List<string> {
+            "15 fps",
+            "30 fps",
+            "60 fps"
+        };
```

Comment:
По-умолчанию выбирается первое из списка. И сейчас при запуске настроек, у меня сразу выставлено 15 фпс. Думаю, стоит поменять порядок на обратный
```suggestion
        List<string> options = new List<string> {
            "60 fps",
            "30 fps",
            "15 fps"
        };
```